ads/contentad.js

```diff
@@ -35,9 +35,10 @@ export function contentad(global, data) {
   /* Pass Source URL */
   let sourceUrl = window.context.sourceUrl;
   if (data.url) {
-    const host = window.context.location.host;
+    const s = document.createElement('a');
```

Comment:
Yes, you should call `parseUrl` to replace the anchor element/`s.href` code here.